views/theme/package.json

```diff
@@ -41,7 +41,7 @@
     "@material/toolbar": "^2.3.0",
     "@material/top-app-bar": "^3.1.0",
     "@material/typography": "^3.1.0",
-    "mdl-stepper": "^1.1.6"
+    "mdl-stepper": "^1.2.1"
```

Comment:
isn't this the one we forked? should we point it at our fork?

Reply:
Good call. I forgot we did that!